docs/ucd_sonic.md

```diff
@@ -0,0 +1,18 @@
+# nf-core/configs: UCD Sonic Configuration
+
+The nf-core pipeline [sarek](https://nf-co.re/sarek) has been successfully tested on [Sonic HPC] (https://www.ucd.ie/itservices/ourservices/researchit/researchcomputing/sonichpc) using this config.
+
+This Sonic config offers very minimal options to users but should allow the use of all nf-core pipelines.
+
+## sabtch_nxf_creator
+To run a pipeline, you should use the [sbatch_nxf_creator](https://github.com/brucemoran/sbatch_nxf_creator) method.
```

Comment:
This seems rather specific/possibly not generalisable to other users and the way they want to run nexfllow.., is it really necessary? Why would you want to process individually for example and enforce a specific file structure

I would rather describe line 16, and then say you offer a helper script if one wants to do specific what you do.